### DISCUSSION TITLE
graphql result가 어디에 있나요?

### DISCUSSION THREAD

**Discussion by chaekite** [changdoc/naver-vaccine-macro#71] [Q&A]
예약 신청 페이지 이동 목록 생성 방법에 보면 매크로 제일 하단에 graphql result에 response를 붙여넣기 하라고 되어있던데 그게 어디에 있나요? 제가 컴맹이라... 친절하게 설명해주시면 감사하겠습니다!

**Comment by changdoc** [comment]
https://github.com/changdoc/naver-vaccine-macro/issues 이곳 들어가보시면 4개의 가이드가 있는데 한번 살펴보세요~

**Reply by chunglok32** [reply]
죄송하지만 여기에 질문을 올려봅니다.
 https://github.com/changdoc/naver-vaccine-macro/issues/4 
에 보면 아래와 같이 되어 있는데," orgCd와 sid 자리"가 어디를 예기하시는 지요?

> vaccineOrganizationCode값과 id값을

> 각각 orgCd와 sid 자리에 복사하여 넣으면
